### PR TITLE
Image rescaling

### PR DESCRIPTION
**Pull request by wolski0420**

### CODE REVIEW

File: rescaling.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import matplotlib.pyplot as matplot
+import numpy as np
+import os
```
Comment (j-adamczyk) [inline rescaling.py:3]: Order of imports should be:
```
# Python built-in modules

# 3rd party libraries

# project libraries
```

File: rescaling.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    """
+
+    # name of temporary file
+    filename = "rescaled_img.jpg"
+
+    # the only way to change DPI I've found is to save image using matplotlib.pyplot.imsave() function
+    # so, I'm saving with 300 DPI, then loading again and deleting temporary file
+    matplot.imsave(filename, image, dpi=300)
+    rescaled_img = scikit.imread(filename)
+    os.remove(filename)
```
Comment (j-adamczyk) [inline rescaling.py:32]: You can use `StringIO` or `BytesIO` buffers from the `io` built-in module. They act as temporary files in RAM, allowing programmer to use objects that require file interface entirely in RAM. It's faster than disk operations.

File: rescaling.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    """
+
+    # reading image with scikit image function
+    img = scikit.imread(filename)
+
+    # rescaling process
+    rescaled_img = rescale(img)
+
+    # saving image to file with scikit image function
+    scikit.imsave(f"{filename.split('.')[0]}_rescaled.jpg", img_as_ubyte(rescaled_img))
```
Comment (j-adamczyk) [inline rescaling.py:60]: Remove for PR

**Review by j-adamczyk — CHANGES_REQUESTED** [review]

**Comment by wolski0420** [comment]
I made changes as you suggested. I had to replace matplotlib with PIL because of loading np.ndarray image to matplotlib problem - matplotlib probably requires more complicated structure than a typical array in numpy

**Review by j-adamczyk — APPROVED** [review]

**Review by konrrad — APPROVED** [review]

**Pull request merged**